pkg/tsdb/elasticsearch/testdata_response/logs.a.golden.jsonc

```diff
@@ -281,124 +281,11 @@
-              "lvl": "error",
-              "nested_field.internal.nested": "value2",
-              "shapes": [
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "square"
-                }
-              ],
-              "testtime": "09/02/2023"
-            },
-            {
-              "abc": "def",
-              "counter": 79,
-              "float": 53.323706427230455,
-              "is_true": true,
-              "label": "val1",
-              "line": "log text  [894867430]",
-              "location": "-38.27341566189766, -23.66739642570781",
-              "lvl": "info",
-              "nested_field.internal.nested": "value3",
-              "shapes": [
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "square"
-                }
-              ],
-              "testtime": "09/02/2023"
-            },
-            {
-              "abc": "def",
-              "counter": 78,
-              "float": 82.72012623471589,
-              "is_true": false,
-              "label": "val1",
-              "line": "log text  [478598889]",
-              "location": "12.373240290451287, 43.265493464362024",
-              "lvl": "info",
-              "nested_field.internal.nested": "value4",
-              "shapes": [
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "square"
-                }
-              ],
-              "testtime": "09/02/2023"
-            },
-            {
-              "abc": "def",
-              "counter": 77,
-              "float": 35.05784443331803,
-              "is_true": false,
-              "label": "val3",
-              "line": "log text  [526995818]",
-              "location": "-31.524344042228194, -32.11254790120572",
-              "lvl": "info",
-              "nested_field.internal.nested": "value5",
-              "shapes": [
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "triangle"
-                },
-                {
-                  "type": "square"
-                }
-              ],
-              "testtime": "09/02/2023"
-            }
+            "{\"abc\":null,\"counter\":81,\"float\":10.911972180833306,\"is_true\":true,\"label\":\"val3\",\"line\":\"log text  [106619125]\",\"location\":\"-42.73465234425797, -14.097854057104112\",\"lvl\":\"info\",\"nested_field.internal.nested\":\"value1\",\"shapes\":[{\"type\":\"triangle\"},{\"type\":\"triangle\"},{\"type\":\"triangle\"},{\"type\":\"square\"}],\"testtime\":\"09/02/2023\"}",
```

Comment:
I understand here is the _source change from json to string, right?

Reply:
yes